app/javascript/driving/helpers/helpers.js

```diff
@@ -38,4 +40,23 @@ const helpers = {
   },
 };
 
+export const RidePropTypes = PropTypes.shape({
+  additional_passengers: PropTypes.number,
+  from_address: PropTypes.string.isRequired,
+  from_city: PropTypes.string.isRequired,
+  from_zip: PropTypes.string.isRequired,
+  name: PropTypes.string,
+  special_requests: PropTypes.string,
+  status: PropTypes.oneOf([
+    'driver_assigned',
+    'picked_up',
+    'waiting_assignment',
+    'waiting_acceptance',
```

Comment:
For after this PR: let's get these into some sort of enum, if only just a module that exports a frozen array and object:
```js
// enum/ride_status.js
export default const statusMap = {
  …
  driverAssigned: 'driver_assigned',
  pickedUp: 'picked_up',
  …
}

export const allStatuses = [
  …
  statusMap.driverAssigned, 
  statusMap.pickedUp,
  …
]
```